types/database/raw_values.d.ts

```diff
@@ -102,7 +102,7 @@ type RawValue =
   | Group
   | GroupChannel
   | GroupTeam
-  | GroupMembership
+  | {group_id: string; user_id: string} // GroupMembership
```

Comment:
Why this change?

Reply:
same question

Reply:
`GroupMembership` from the API returns `UserProfile[]` from `/user/{userId}/groups`, but in our case - we want both the `group_id` and `user_id` (or `user.id`) to make the association. We can get `user.id` for the `user_id` part, but we can't get `group_id` without sidestepping a lot of the type safety, hence why it's easier to not use `GroupMembership`.

I can reassign `GroupMembership = {group_id: string; user_id: string}` but then it would contradict what the API returns.